dart/test/mocks/mock_hub.dart

```diff
@@ -105,9 +107,6 @@ class MockHub implements Hub {
     _isEnabled = false;
   }
 
-  @override
-  void configureScope(callback) {}
```

Comment:
`clone, `lastEventId`, `startTransaction`,  `startTransactionWithContext`, also do nothing.
There's a return object because it's needed to compile, but they don't do anything else.
I believe other mocks should have the very same type of methods that can be removed.